docs/content/docs/start/tutorial.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Tutorial: Create a Fluid Framework application'
+title: 'Tutorial: Create a Fluid app'
```

Comment:
```suggestion
title: 'Tutorial: Hello world walkthrough'
```

Reply:
going with capital W since it's more of a title than a sentence